products/crop_mask_product.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: crop_mask
+description: prediction based s2 l2a gm_tmad, NDVI, LAI, MNDWI, rainfall, slope
+metadata_type: eo3
+
+license: CC-BY-4.0
+
+metadata:
+  product:
+    name: crop_mask
+
+measurements:
+  - name: MASK
+    aliases: ['crop_mask']
+    dtype: uint8
+    nodata: 255
+    units: '1'
+
+  - name: PROB
+    aliases: ['crop_prob']
+    dtype: uint8
+    nodata: 255
+    units: '1'
+
+storage:
+  crs: EPSG:6933
```

Comment:
Let's use lower-case for epsg to be compatible with ARD data in AU.

Reply:
sure.